bindings/cpp/include/opendal.hpp

```diff
@@ -147,6 +147,13 @@ class Operator {
    */
   void remove(std::string_view path);
 
+  /**
+   * @brief Remove all files and directories recursively
+   *
+   * @param path The path of the directory to remove recursively
+   */
+  void remove_all(std::string_view path);
```

Comment:
This API will be removed in the future. Let's just don't add them if not requested.